tests/yast2_gui/yast2_firewall_set_default_zone.pm

```diff
@@ -32,9 +31,11 @@ sub run {
     save_screenshot;
     $testapi::distri->get_firewall()->accept_change();
     assert_screen 'generic-desktop';
-    select_serial_terminal();
+    select_console 'root-console';
+    systemctl 'restart firewalld', timeout => 200 if (script_output("grep 'FlushAllOnReload.*no' /etc/firewalld/firewalld.conf") =~ 'no');
```

Comment:
if `script_output` fails you don't get output to compare with a regex, but you are doing twice the same thing, grepping 'no' and then a regex with 'no'.
better use `if script_run($cmd) ==0)`